src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne()
+    private Movie movie;
+    @ManyToOne()
```

Comment:
```suggestion
    @ManyToOne
```
- let's set correct column name cinema_hall_id